packages/frontend/components/primitives/image-with-fallback/component.tsx

```diff
@@ -0,0 +1,25 @@
+import { useEffect, useState } from 'react';
+import Image from 'next/image';
```

Comment:
It's better to use next/future/image since this component will be deprecated

Reply:
done